### PR TITLE
centos compatibility

### PR DESCRIPTION
**Pull request by merryech0**
- compatibility with centos, apache (not checked)
- separate installation of extension and dev-tools
- more cleaner way to install extension
- more cleaner way to install dev-tools
- install bash completion for dev-tools

### CODE REVIEW

File: provision/shell/install_phalcon.sh [hunk 1]
```diff
@@ -27,42 +32,45 @@ else
...
-        ln -s /etc/php5/mods-available/phalcon.ini /etc/php5/fpm/conf.d/phalcon.ini
+    # write ini config
+    if ! (php --ri phalcon &>/dev/null); then
+        if [ -d "/etc/php5.d" ]; then
+            echo 'extension=phalcon.so' > /etc/php5.d/phalcon.ini
+        elif [ -d "/etc/php.d" ]; then
+            echo 'extension=phalcon.so' > /etc/php.d/phalcon.ini
+        elif [ -d "/etc/php5/mods-available" ]; then
+            echo 'extension=phalcon.so' > /etc/php5/mods-available/phalcon.ini
+            [ -d '/etc/php5.d/cli' ] && ln -s /etc/php5/mods-available/phalcon.ini /etc/php5/cli/conf.d/phalcon.ini
```
Comment (slogsdon) [inline provision/shell/install_phalcon.sh]: Looks like there's a typo here and the next two lines in the tests. `/etc/php5.d` should be `/etc/php5`

**Comment by slogsdon** [comment]
Thanks for contributing! I made a comment on a section of the `install_phalcon.sh` script that I had trouble with when testing your changes, but otherwise, everything looks great. If you want to update that file, I'll be glad to merge your changes.

File: provision/shell/install_phalcon.sh (hunk 1 above)
Comment (merryech0) [inline provision/shell/install_phalcon.sh]: Yes, sorry.
I'll made a fix

**Comment by slogsdon** [comment]
Thanks for the contribution @i-ekho!

**Comment by merryech0** [comment]
not at all ;)

**Pull request merged**